src/node/ledgersecrets.h

```diff
@@ -118,19 +123,20 @@ namespace ccf
         auto search = secrets_map.find(v);
         if (search != secrets_map.end())
         {
-          throw std::logic_error(
-            "Cannot restore secrets that already exist: " + std::to_string(v));
+          throw std::logic_error(fmt::format(
+            "Cannot restore ledger secret that already exist: ", v));
```

Comment:
Minor: `already exists`

Reply:
Good spot! There was some confusion between "secrets" and "secret" in this file since the network identity used to be part of a "secret" entry as well (until https://github.com/microsoft/CCF/pull/674). I think I've fixed the ambiguities now.